ledger-api/grpc-definitions/com/daml/ledger/api/v1/completion.proto

```diff
@@ -24,7 +24,7 @@ message Completion {
   // Optional
   google.rpc.Status status = 2;
 
-  // The transaction_id of the transaction that resulted from the command with command_id.
+  // The transaction_id of the transaction, or the update_id of the transfer that resulted from the command with command_id.
   // Only set for successfully executed commands.
   // Must be a valid LedgerString (as described in ``value.proto``).
   // Optional
```

Comment:
Is it worth renaming `transaction` to something different to indicate that it is not necessarily about a transaction anymore ?

Reply:
I think it is worth for 3.0 where we also did that: `update_id` see in https://github.com/digital-asset/daml/pull/15471
For this increment I would keep it as close as possible to the original. (For example this rename can cause already issues with protobuf code generation, AFAIR current backwards compatibility does not allow it neither)

Reply:
as discussed we go back to the update_id approach, will adapt the pr soon